### DISCUSSION TITLE
Noob Help: tuya-cli wizard: There was an issue fetching that device. Make sure your account is linked and the ID is correct.

### DISCUSSION THREAD

**Discussion by zeroalpha-brisbane** [codetheweb/tuyapi#539] [General]
Just want to post some help for other noobs out there like me. I struggled with all the jargon and navigating Tuya's website. If you know what to look for its really easy to setup. I hope this helps someone else setup their tuyapi.

If you run `tuya-cli wizard` and get this error: `There was an issue fetching that device. Make sure your account is linked and the ID is correct.`

Make sure you enter the correct information. Tuya had changed some names since the tutorial was written

If you have successfully linked your tuya app with your developer account you should be able to add a new smart device to your app. and it will show up in your developer account as you add them.

Get your API Key & API Secret on this page:

![tuya0](https://user-images.githubusercontent.com/92899951/139789050-53396414-bde5-4f0e-b6b4-8cda3306e540.PNG)

Get a virtual ID here: Doesn't matter which virtual ID you use it just wants one:

![tuya1](https://user-images.githubusercontent.com/92899951/139789103-a1efe1e1-664b-438d-94a6-289a685d1045.png)

Now run `tuya-cli wizard` and enter the details

![tuya2-node-red](https://user-images.githubusercontent.com/92899951/139789113-4bc6f27d-2fb8-49ad-9442-3759cace8632.PNG)

BAM! You should get all your keys! I know this is only a small part of the setup instructions but I hope it helps other noobs.

**Comment by p6002** [comment]
How to fix it because it used to work normally and now I wanted to connect a new device no longer works. Everything activated, in the application I added the device:
There was an issue fetching that device. Make sure your account is linked and the ID is correct.

**Comment by evilsean** [comment]
**Make a new Tuya developer account** and repeat all the steps if the 1 month trial expires.

**Comment by p6002** [comment]
It does not matter any trial, there were problems with the servers back then.

**Reply by JonBuhl** [reply]
so did it eventually work for you again without you doing anything?
